recipes/python-chromedriver-binary/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "python-chromedriver-binary" %}
+{% set version = "2.36.0" %}
+{% set sha256 = "b57abdd7bdfb1bdc8070aa2451d2945cbfa70389297767ebf540889fbd35d91a" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/danielkaiser/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
```

Comment:
Not sure what conda build does when it finds a `bld.bat`/`build.sh` file and also a `build/script` section, does it run both? Regardless I think you should be explicit and just have `bld.bat`/`build.sh` call `python -m  pip ...`